web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -300,7 +300,8 @@ const ExtraTopComponents = (props: {
                   showErrorsInModal={() => {
                     showErrorsInModal(nonRetryableResourceSearchErrors);
                   }}
-                  advancedSearch={advancedSearch}
+                  // We show the advanced search in TypeToSearchItem.
+                  advancedSearch={undefined}
```

Comment:
Should this be backported to v14?

Reply:
I don't remember if v14 has those toggles or not. 🥴

Reply:
No, it hasn't.